pkg/controller/direct/spanner/instance_controller.go

```diff
@@ -229,7 +229,8 @@ func (a *SpannerInstanceAdapter) Update(ctx context.Context, updateOp *directbas
 		updateMask.Paths = append(updateMask.Paths, "labels")
 	}
 
-	if !reflect.DeepEqual(resource.DefaultBackupScheduleType, a.actual.DefaultBackupScheduleType) {
+	// if defaultBackupScheduleType is not set in spec, the field become unmanaged.
```

Comment:
I feel like this needs to be a separate PR, it seems very unrelated but also very significant...